src/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -1,16 +1,16 @@
-import { anything, capture, instance, mock, verify, when } from '@typestrong/ts-mockito'
+import { anything, instance, mock, verify, when } from '@typestrong/ts-mockito'
 import { Rundown } from '../../../model/entities/rundown'
 import { RundownEventEmitter } from '../interfaces/rundown-event-emitter'
 import { RundownRepository } from '../../../data-access/repositories/interfaces/rundown-repository'
 import { TimelineRepository } from '../../../data-access/repositories/interfaces/timeline-repository'
 import { TimelineBuilder } from '../interfaces/timeline-builder'
-import { RundownEventBuilder } from '../interfaces/rundown-event-builder'
+import { RundownEventBuilder } from '../../../presentation/interfaces/rundown-event-builder'
 import { ActiveRundownException } from '../../../model/exceptions/active-rundown-exception'
-import { RundownEventType } from '../../../model/enums/rundown-event-type'
+import { RundownEventType } from '../../../presentation/enums/rundown-event-type'
 import { RundownTimelineService } from '../rundown-timeline-service'
 import { CallbackScheduler } from '../interfaces/callback-scheduler'
 import { EntityMockFactory } from '../../../model/entities/test/entity-mock-factory'
-import { RundownDeletedEvent } from '../../../model/value-objects/rundown-event'
+import { RundownDeletedEvent } from '../../../presentation/value-objects/rundown-event'
```

Comment:
This is not allowed to know about the Presentation layer

Reply:
I have removed the imports :+1: